src/main/groovy/life/qbic/registration/Provenance.groovy

```diff
@@ -0,0 +1,27 @@
+package life.qbic.registration
+
+import groovy.json.JsonSlurper
+
+class Provenance {
+
+    final String origin
+    final String user
+    final String measurementID
+    final List<String> history
+
+    Provenance(String origin, String user, String measurementID, List<String> history) {
```

Comment:
I think some JD would help in understanding what this class is supposed to represent 🤔